ProjFS.Mac/PrjFSKext/PrjFSKext/Info.plist

```diff
@@ -15,11 +15,9 @@
 	<key>CFBundlePackageType</key>
 	<string>KEXT</string>
 	<key>CFBundleShortVersionString</key>
-	<string>1.0</string>
+	<string>0.1.1</string>
 	<key>CFBundleVersion</key>
-	<string>1</string>
-	<key>NSHumanReadableCopyright</key>
-	<string>Copyright © Microsoft</string>
+	<string>0.1.1</string>
```

Comment:
Since we're changing the interface in the kernel, I think this is worthy of a version number change. Since we're not released yet, I thought 1 was an incorrect number to start from so I chose 0.1.1 as this is a 'minor' change on top of 0.1.

I'm not very picky on what numbering scheme we choose here, but I do think we need to bump when we make a breaking change to our interfaces :)